spec/controllers/home_controller_spec.rb

```diff
@@ -30,6 +30,22 @@
     @test_settings.should_receive(:home_page_content).at_least(:once).and_return(content)
     @test_settings.stub(:name).and_return("Test Settings")
 
+    #
+    # Ensure this mock responds to "custom_search_path"
+    #
+    # See fix for PT 144299359: Figure out how to run non-javascript 
+    #                           cucumber tests when doing development 
+    #                           using docker.
+    #
+    # Note PT #144866753: Portal rspec test fails in docker.
+    # It appears some test configurations are not using the 
+    # appropriate mock. Ensure tests with this spec use this mock
+    # and ensure this mock responds to this message. 
+    # Responding with nil is fine as callers should check for that
+    # and handle it appropriately.
+    #
```

Comment:
This comment should be above the line `@test_settings = mock("settings")`.  The two issues apply to all the tests that are expecting to use this mock. Sometimes they do and sometimes they don't.